### DISCUSSION TITLE
Missing images on gh-pages

### DISCUSSION THREAD

**Discussion by arahmitz** [nunocoracao/blowfish#1243] [Q&A] [answered]
Hey,

I've set up gh-pages today and I am having trouble with static images. Everything works on localhost, but when I try to push on github actions it seems like my link breaks.

Link to repo: https://github.com/arahmitz/adamblogstuff
Link to page: https://arahmitz.github.io/adamblogstuff/

Seems like the image is parsed correctly, as you can connect to https://arahmitz.github.io/adamblogstuff/images/ballrig_blender_1.png just fine, it just doesn't render them in posts.

I've tried changing paths from (/images/*) to (images/*) but seems like it doesn't work.

**Comment by nunocoracao** [comment]
Did you define the baseurl correctly?

**Reply by arahmitz** [reply]
Seems like baseURL should be alright
```
baseURL = "//arahmitz.github.io/adamblogstuff/"
```

Basically avoiding the protocol (as someone pointed that it might sometimes break stuff) and with / at the end, that's why I've changed image paths to start without.

using full https base url didn't really change anything.

**Comment by arahmitz** [comment]
I think I've found the cause of this problem. 

I am not sure why, but when I was trying to link images to static folder (`images/imagename.jpg`) it tries to path things under content/posts/postfolder/images path instead of using the static folder instead. At this point I'll try to run a job later while moving around rest of the content to be properly slotted with guidelines.

**Reply by arahmitz** [reply, marked as answer]
So... I feel stupid but I've fixed it. 

I used the full path to images like `(/reponame/images/image.jpg)`
